src/main/resources/schema.sql

```diff
@@ -1,8 +1,16 @@
-CREATE TABLE reservation
+CREATE TABLE time
 (
-    id      BIGINT       NOT NULL AUTO_INCREMENT,
-    name    VARCHAR(255) NOT NULL,
-    date    VARCHAR(255) NOT NULL,
-    time    Timestamp NOT NULL,
+    id   BIGINT       NOT NULL AUTO_INCREMENT,
+    time VARCHAR(255) NOT NULL,
     PRIMARY KEY (id)
 );
+
+CREATE TABLE reservation
+(
+    id   BIGINT       NOT NULL AUTO_INCREMENT,
+    name VARCHAR(255) NOT NULL,
+    date VARCHAR(255) NOT NULL,
+    time_id BIGINT,                           // TODO: 수정
```

Comment:
Sql의 주석은 `--`로 표기해요.
잘못된 주석 표기로 Sql 문법 오류가 나서 테스트 실행이 안되고 있어요ㅠ
```suggestion
    time_id BIGINT,                           -- TODO: 수정
```